Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Используем официальный образ Python
+FROM python:3.8-slim AS build
```

Comment:
The alias 'AS build' is not used anywhere in the Dockerfile. If you don't plan to use it for multi-stage builds, you can remove it.